src/webgpu/util/f32_interval.ts

```diff
@@ -60,6 +60,16 @@ export class F32Interval {
     return this.begin <= i.begin && this.end >= i.end;
   }
 
+  /** @returns if any subnormal values are contained within the interval.
+   *           This is used to check if an interval contains zero or valyes that
+   *           may be flushed to zero.
+   */
+  public containsSubnormals(): boolean {
```

Comment:
This can be simplified to via De Morgan's Law to be end >= min && begin <= max. As written it is cleared to me what the logic is, `test of the entire interval it above or below the subnormals, otherwise it has to intersect with them`.